src/minimax_server/router.clj

```diff
@@ -0,0 +1,18 @@
+(ns minimax-server.router
+  (:import [java.util.function Function]
+           [scarvill.httpserver.response ResponseBuilder])
+  (:require [clojure.string :as str]))
+
+(defn response-with-body [body]
+  (.build (.setBody (new ResponseBuilder) (byte-array (map byte (str body))))))
+
+(defn get-current-player [request]
+  (.getParameterValue request "current_player"))
+
+(defn get-board [request]
+  (str/split (.getParameterValue request "board") #","))
+
+(defn minimax-router [get-move]
+  (proxy [Function] []
+    (apply [request]
+           (response-with-body (get-move (get-current-player request) (get-board request))))))
```

Comment:
We decided not to worry about malformed params because our server just returns a 200 with an empty body in such cases. Would you recommend that we return a 400 Bad Request or something? 

I like the idea about creating a Jetty or Ring style application. I briefly looked those up, and it would be a nice way to go, but I think that would be something for the next iteration. A new story perhaps?